tests/Aspire.Dashboard.Components.Tests/Pages/MetricsTests.cs

```diff
@@ -121,6 +121,7 @@ public void InitialLoad_HasSessionState_RedirectUsingState()
     }
 
     [Fact]
+    [ActiveIssue("https://github.com/dotnet/aspire/issues/8299")]
```

Comment:
Looks like this issue is closed now, so perhaps this can get re-enabled? :smiley:

Reply:
done